code/modules/clothing/under/accessories.dm

```diff
@@ -1,230 +1,261 @@
-/obj/item/clothing/tie //Ties moved to neck slot items, but as there are still things like medals and armbands, this accessory system is being kept as-is

```

Comment:
Use the same as it was before(can easily check the upstream's version on notepad++). Changing the line ending fucks up with git blame, which can later be a pain for the history.